Gemfile

```diff
@@ -20,7 +20,7 @@ if allow_local && File.exist?('../openstudio-model-articulation-gem')
   # gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'develop'
   gem 'openstudio-model-articulation', path: '../openstudio-model-articulation-gem'
 else
-  gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'develop'
+  gem 'openstudio-model-articulation', github: 'NREL/openstudio-model-articulation-gem', branch: 'Edits'
```

Comment:
@tanushree04 this is just for temporary debugging right https://github.com/NREL/openstudio-model-articulation-gem/commit/38003aec360cf26ba05e68f936806da3b8ed216c ?  If so, I will just delete that branch of the model articulation repo

Reply:
There are also some previous commits for shortening file names .